dbqueries.py

```diff
@@ -17,20 +19,26 @@
             ORDER BY REGEXP_SUBSTR(term, '^\\\\D+') DESC,
             NULLIF(REGEXP_SUBSTR(term, '.+', REGEXP_INSTR(term, '\\\\d+')+4), '') ASC NULLS FIRST, 
                 REGEXP_SUBSTR(term, '\\\\d+') DESC;
-        """
+        """,
+        'checks': {}
     },
     'unizin_metadata': {
-        'index': '',
         'output_file_name': 'unizin_metadata.csv',
         'data_source': 'UDW',
         'query_name': 'Unizin Metadata',
         'type': 'standard',
         'query': """
             SELECT * FROM unizin_metadata;
-        """
+        """,
+        'checks': {
```

Comment:
These are pretty cool ideas, I wonder if we'd want to refactor the checks out of the individual tables though so they can be re-used. Also I'm thinking we might want a check here that is like less_than_five_days and return a RED, but maybe that's a more generic less_than_x_days. But how would the "x" be passed in? Maybe just hardcoding 5 is enough.

Reply:
I can imagine a `CHECKS` dictionary comes before `QUERIES` and we could just reference the right `check` dictionary where necessary. We could also have other `input` key-pair somewhere, maybe where the `check` dictionary is referenced. Lambda functions can take multiple inputs (e.g. `lambda x, y ...`). Seems cool but perhaps unnecessary now, as we have so few tests?

Reply:
I think it's fine for now, just commit this and we'll roll this out. We're going to have to add some additional checks in a future phase anyway when we hook up a database.